src/main/java/tech/gruppone/stalker/server/repositories/UserRepository.java

```diff
@@ -3,13 +3,20 @@
 import org.springframework.data.r2dbc.repository.Modifying;
 import org.springframework.data.r2dbc.repository.Query;
 import org.springframework.data.repository.reactive.ReactiveCrudRepository;
+import reactor.core.publisher.Flux;
 import reactor.core.publisher.Mono;
 import tech.gruppone.stalker.server.model.db.UserDao;
 
 public interface UserRepository extends ReactiveCrudRepository<UserDao, Long> {
 
   public Mono<UserDao> findById(Long id);
 
+  @Query("select * from Users")
+  public Flux<UserDao> findAll();
+
+  @Query("SELECT * FROM User u WHERE u.email = :email")
+  public Mono<UserDao> findByEmail(String email);
```

Comment:
in teoria se dichiari `@Table` e `@Column` sul Dao queste query vengono inferite automaticamente da r2dbc (cioè funziona senza l'annotation)